postgresql/resource_postgresql_function_test.go

```diff
@@ -144,6 +145,107 @@ resource "postgresql_function" "increment" {
 	})
 }
 
+func TestAccPostgresqlFunction_FailsCreateOnAlreadyExistingFunction(t *testing.T) {
```

Comment:
Lemme know if you'd prefer this addition (https://github.com/Leafly-com/terraform-provider-postgresql/pull/4/commits/6d7b8b48e63537240b78d25f67edaed60aac58f8) be squashed into the first commit that adds the general functionality